server/acl_casbin_policy_dba.csv

```diff
@@ -81,6 +81,7 @@ p, DBA, /bookmark/user/{userID}, GET_SELF
 p, DBA, /bookmark/{bookmarkID}, DELETE_SELF
 p, DBA, /pipeline/{pipelineID}/stage/{stageID}/status, PATCH
 p, DBA, /pipeline/{pipelineID}/task/all, PATCH
+p, DBA, /pipeline/{pipelineID}/task/rollback, POST
```

Comment:
This limits to rolling back a single task. The other option is to use issue API for entrance. That's the multiplexer for creating issues. Also, it doesn't feel right to use task API to create an issue.

@tianzhou WDYT?

Reply:
Better to put it in issue.

According to our data model, Issue has knowledge of pipeline, while pipeline (thus stage, task) does not have knowledge of issue.

Reply:
Moved to under issue. PTAL @d-bytebase @tianzhou